### ISSUE TITLE
"Hot Right Now" reward cannot be claimed 

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-app#170]
## The Issue
After watching 3 of the featured videos, the claim function on Hot Right Now still says "You haven't watched a feature video yet
### Steps to reproduce
1. Watch feature vidoes
2. Go to wallet > Rewards
3. Click Claim Reward on Hot Right Now

### Expected behaviour
reward should be redeemed

### Actual behaviour
Told I have not watched a video

## System Configuration

 

- LBRY Daemon version:
- LBRY App version: 0.11.8
- LBRY Installation ID:
- Operating system: Win

**Comment by tzarebczan** [comment]
@lyoshenka I did open an issue for what we discussed on slack. The request gets a 400 response, this is the data:
Access-Control-Allow-Methods:GET, POST, OPTIONS
Access-Control-Allow-Origin:file://
Connection:keep-alive
Content-Length:94
Content-Type:application/json; charset=utf-8
Date:Thu, 01 Jun 2017 22:20:41 GMT
Server:nginx/1.10.1
X-Content-Type-Options:nosniff
Request Headers
view source
Accept:*/*
Accept-Encoding:gzip, deflate
Accept-Language:en-US
Connection:keep-alive
Content-Length:153
Content-type:application/x-www-form-urlencoded
Host:api.lbry.io
Origin:file://
User-Agent:Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) LBRY/0.11.9 Chrome/56.0.2924.87 Electron/1.6.10 Safari/537.36
Form Data
view source
view URL encoded
app_id:3vbBV2oD6JvWcGsfWrhdfwKhqSD2bcqeZbMST4EGwXCS49Xf4R9tUaozmM7ZvAZ13B
reward_type:featured_download
wallet_address:bUPgTKAzzUZsvAgWuwYv3cbcPbJB8rBngR

**Comment by lyoshenka** [comment]
Fixed :-)

**Comment by tzarebczan** [comment]
Confirmed working on Windows where I tried a bunch of the featured content. On my linux VM, I just watched the Cryptoverse episode and tried to claim...same issue where it says the featured content wasn't downloaded. Is it enough to watch a single video, is there a delay? 

app_id:7PpaKawsC47yjrfPWjnythmJ9bLUk2qw1NsVAcaB6PUaVM9zujhYv3B9LwbdDkQp7L
reward_type:featured_download
wallet_address:bHSTQX1iEHSJQvM76jCusHYsTknNiSPXpo

**Comment by lyoshenka** [comment]
That cryptoverse episode is not one of the ones that's included in the reward. Does the app make it look like it is? That's probably an issue with the app and how it presents that info. Can you include a screenshot please?

Only `@TheCryptoverse/tc-G8w7lxlk2p0` is eligible for that reward.

**Comment by tzarebczan** [comment]
There is nothing in the app that tells you whether or not the content is featured, I got that channel from the email (which lists channels only).  Hmm, maybe there should be an indicator on the content itself too to make that more apparent? 

Need to double check if it launched the channel, or if I got that episode from the discover page. There may be 2 different paths to the claim. Will check tonight after work and let you know!
